app/megatron/commands/command_actions.py

```diff
@@ -370,4 +370,21 @@ def _change_pause_state(
         ActionType.POST_MESSAGE, {"channel": channel, "message": msg}
     )
     integration_connection.take_action(message_action)
+
+    paused_phrase = (
+        "Hey, there! I've been paused so a support agent can talk with you. I'll let you know when I'm back online"
+        if pause
+        else "Hey, there! I've been unpaused and I'm back online to help you with everything I can. "
+    )
+    user_msg = {
+        'text': paused_phrase,
+        'attachments': [
+            {
+                "text": "",
+                "footer": f"Executed by Teampay.",
```

Comment:
Can we make this footer show the person who paused the bot? Like it shows on messages

Reply:
@gonzalomolbar ?

Reply:
Damn, sorry, this repository doesn't notify me, I don't know why... 
I thought of it when I didi it, although I didn't implemented at the end finally, I can't tell you why, maybe because I forgot to be honest. Will check it out right now

Reply:
Done! I don't control the `get command` because in all the calls to that method the user has to be formally created to don't give a previous error so is kinda impossible to don't get the user that the request brings.
It's the only doubt that I can have regarding this.